framework/math/SpatialDiscretization/SpatialDiscretizatoin.cc

```diff
@@ -310,14 +310,14 @@ double SpatialDiscretization::CartesianSpatialWeightFunction(
 double SpatialDiscretization::CylindricalRZSpatialWeightFunction(
   const chi_mesh::Vector3& point)
 {
-  return point[0];
+  return 2.0 * M_PI * point[0];
```

Comment:
is this used inn lbs cylindrical? I think not, we have to have be careful about the convention used there when we look at it later nad possibly change it

Reply:
It does get used there. And it makes no difference since it applies a constant to the whole equation. Where this makes a difference is when one computes volumetric quantities like total power from heat generation rate. The 2pi is necessary to compute the correct volume.

Reply:
I encountered it in a post-processor